testrail.example.cfg

```diff
@@ -0,0 +1,10 @@
+[API]
+url = https://instance.testrail.net/
```

Comment:
i know this is just an example, but in reality we're going to have to eventually pull all these configs from the environment, is that possible?

Reply:
Hmm.. I don't think the pytest-testrail plugin uses env vars. I'll have to double check.

Reply:
Yea, pytest-rail out of the box doesn't support environment variables. I'll need to add those.

Reply:
ok, in CI we can specify them via command line, that'll be easier

Reply:
👍

Reply:
I removed the file and added instructions in the readme to create a file if someone wanted to run it locally and create their own run.